plugins/3scale-backend/dist-dynamic/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@janus-idp/backstage-plugin-3scale-backend-dynamic",
-  "version": "1.1.2",
+  "version": "1.3.0",
```

Comment:
We should not touch other plugins, only `orchestrator*`. Please revert the changes on them.

Reply:
Okay sure. I think yesterday while try different `yarn install` commands they got auto changed.